### ISSUE TITLE
'parseResult' problem with Windows OS

### ISSUE THREAD

**Issue by moelders** [jmendiara/node-closure-linter-wrapper#6]
When executing the _closure-linter-wrapper_, the results of the _stdout_ are splitted by new line character. Under _Windows OS_ these lines finish with a `\r` character and the regular-expressions to check the results are not passed.

I think you can update the two problematic regular-expressions adding, before the `$` (match at the end of input) the `\s` wildcard (equivalent to [ \t\r\n\v\f]).

On the other hand there is another issue with the `skippingRegex` regular-expression when more than nine files are skipped the regular-expression does not match because it should be `\\d+`.

The results:

``` javascript
successRegex = new RegExp('^([\\d]+) file.*no errors found.?\\s?$')
skippingRegex = new RegExp('^Skipping \\d+ file\\(s\\)\\.?\\s?$')
```

Thanks.
